casper/src/test/scala/coop/rchain/casper/sync/BlockRetrieverRequesAllSpec.scala

```diff
@@ -189,10 +175,29 @@ class BlockRetrieverRequesAllSpec extends FunSpec with BeforeAndAfterEach with M
             // when
             blockRetriever.requestAll(timeout).runSyncUnsafe()
             // then
-            val requestedBlocksMapAfter = currentRequests.get.runSyncUnsafe()
-            requestedBlocksMapAfter.size should be(0)
+            val (_, msg) = transportLayer.getRequest(0)
+            toHasBlockRequest(msg).hash should be(hash)
+            transportLayer.requests.size should be(1)
           }
         }
+        it(
+          "should remove the entry from the requested block lists when block is in casper buffer and after timeout"
+        ) {
+          // given
+          val waitingList = List.empty[PeerNode]
+          val requested = RequestState(
+            timestamp = timedOut,
+            inCasperBuffer = true,
```

Comment:
Just curious. Why the `inCasperBuffer=true` is added

Reply:
It is not required and wont influence test case, looks like leftover from other case.  I removed it.